components/transport_profiles/Gemfile

```diff
@@ -0,0 +1,17 @@
+source 'https://rubygems.org'
+
+# Declare your gem's dependencies in transport_profiles.gemspec.
+# Bundler will treat runtime dependencies like base dependencies, and
+# development dependencies will be added by default to the :development group.
+gem "transport_gateway", path: File.expand_path("../../transport_gateway", __FILE__)
```

Comment:
Can't cross-require engines in gemspec from a relative path.  It sucks, but this is how you gotta do it.